src/garage/torch/policies/stochastic_policy.py

```diff
@@ -55,6 +57,24 @@ def get_actions(self, observations):
                         values of the distribution.
 
         """
+        if not isinstance(observations[0], np.ndarray) and not isinstance(
+                observations[0], torch.Tensor):
+            observations = self._env_spec.observation_space.flatten_n(
+                observations)
+
+        # frequently users like to pass lists of torch tensors or lists of
```

Comment:
is there a reason you added this feature? i thought this was a backport

Reply:
but this wasn't necessary to pass the torch tests? if you are going to add the feature to torch, please also add the tests to torch.